doc/source/drivers/vector/pmtiles.rst

```diff
@@ -0,0 +1,337 @@
+.. _vector.pmtiles:
+
+================================================================================
+PMTiles -- ProtoMaps Tiles
+================================================================================
+
+.. versionadded:: 3.8
+
+.. shortname:: PMTiles
+
+This driver supports reading and writing `PMTiles <https://github.com/protomaps/PMTiles>`__
+datasets containing vector tiles, encoded in the MapVector Tiles (MVT) format.
+
+PMTiles is a single-file archive format for tiled data. A PMTiles archive can
+be hosted on a commodity storage platform such as S3, and enables low-cost,
+zero-maintenance map applications that are "serverless" - free of a custom tile
+backend or third party provider.
+
+This driver is compatible with all GDAL
+:ref:`network-based virtual file systems <network_based_file_systems>`
+
+The SRS is always the `Pseudo-Mercator <https://en.wikipedia.org/wiki/Web_Mercator_projection>`__
+(a.k.a Google Mercator) projection, EPSG:3857.
+
+The supported datasets must contain a JSON metadata document
```

Comment:
This should be OK but is technically not part of the PMTiles spec yet - see [3.1 PR](https://github.com/protomaps/PMTiles/pull/123) since I have been waffling on the exact definition of `fields` here.

Reply:
I have harmonized this in [3.1](https://github.com/protomaps/PMTiles/blob/main/spec/v3/spec.md#metadata) with [TileJSON](https://github.com/mapbox/tilejson-spec/tree/master/3.0.0#33-vector_layers), so there's no surprises here.